homeassistant/components/huawei_lte/__init__.py

```diff
@@ -275,7 +275,7 @@ def logout(self) -> None:
         except Exception:  # pylint: disable=broad-except
             _LOGGER.warning("Logout error", exc_info=True)
 
-    def cleanup(self, *_) -> None:
+    def cleanup(self, *_: Any) -> None:
```

Comment:
`self.cleanup` gets passed to `HomeAssistant.async_add_executor_job` and `EventBus.async_listen_once`, and the relevant para.eter in the first is `Callable[..., T]`, and in the latter it's just `Callable`. As we are not affecting what actually gets passed to this function as parameters, I don't think it would be quite appropriate to make the type hints more specific here -- that would be manifesting internal implementation detail of those functions more than they currently expose in their type hints.

(It would be good to make _those_ type hints more specific if possible, but that's something for a different PR.)